index.bs

```diff
@@ -2630,6 +2630,8 @@ credential.
         The [=user handle=] MUST NOT contain [PII] about the user, such as a username or e-mail address;
         see [[#sctn-user-handle-privacy]] for details.
 
+        Note: the [=user handle=] should not be a constant value across different accounts, even for [=non-discoverable credentials=], because some authenticators always create [=discoverable credentials=]. Thus a constant [=user handle=] would prevent a user from using such an authenticator with more than one account at the [=[RP]=]. Additionally, the empty value is known to have problems in practice so is a bad choice, even if a constant value is desired.
```

Comment:
maybe?
```suggestion
        Note: the [=user handle=] should not be a constant value across different accounts, even for [=non-discoverable credentials=], because some authenticators always create [=discoverable credentials=]. Thus a constant [=user handle=] would prevent a user from using such an authenticator with more than one account at the [=[RP]=]. Additionally,  a [=user handle=] having an empty value is known to be problematic in practice so is a bad choice, even if a constant value is desired.
```